<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MHebrDict</class>
 <widget class="QWidget" name="MHebrDict">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hebrew Dictionary</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resfile.qrc">
    <normaloff>:/new/icons/icons/abgd.png</normaloff>:/new/icons/icons/abgd.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="twInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabReg">
        <attribute name="title">
         <string>re&amp;gular</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="grpMethod">
           <property name="title">
            <string>method</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="rbRegexp">
              <property name="text">
               <string>&amp;regexp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbExact">
              <property name="text">
               <string>&amp;exact</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="CLatCopt" name="txtInput" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabFulltext">
        <attribute name="title">
         <string>fullte&amp;xt (regexp only)</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="CLatCopt" name="txtFulltext" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabByNumber">
        <attribute name="title">
         <string>&amp;by number</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>320</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>&amp;number (1-8674)</string>
               </property>
               <property name="buddy">
                <cstring>spnByNumber</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSpinBox" name="spnByNumber">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>8674</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="cmbItems">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;page</string>
         </property>
         <property name="buddy">
          <cstring>spnPage</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnPage">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;limit</string>
         </property>
         <property name="buddy">
          <cstring>spnLimit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnLimit">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btStart">
         <property name="text">
          <string>s&amp;earch</string>
         </property>
         <property name="icon">
          <iconset resource="resfile.qrc">
           <normaloff>:/new/icons/icons/work.png</normaloff>:/new/icons/icons/work.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="CBookTextBrowser" name="wdgOutput" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="wdgConc" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QToolButton" name="tbFontConcPlus">
           <property name="toolTip">
            <string>larger font</string>
           </property>
           <property name="text">
            <string>larger font</string>
           </property>
           <property name="icon">
            <iconset resource="resfile.qrc">
             <normaloff>:/new/icons/icons/loupe_plus.png</normaloff>:/new/icons/icons/loupe_plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbFontConcMinus">
           <property name="toolTip">
            <string>smaller font</string>
           </property>
           <property name="text">
            <string>smaller font</string>
           </property>
           <property name="icon">
            <iconset resource="resfile.qrc">
             <normaloff>:/new/icons/icons/loupe_minus.png</normaloff>:/new/icons/icons/loupe_minus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblConcTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbActionConc">
           <property name="text">
            <string>actio&amp;n</string>
           </property>
           <property name="icon">
            <iconset resource="resfile.qrc">
             <normaloff>:/new/icons/icons/action.png</normaloff>:/new/icons/icons/action.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="tbConcHide">
           <property name="toolTip">
            <string>hide</string>
           </property>
           <property name="text">
            <string>hide</string>
           </property>
           <property name="icon">
            <iconset resource="resfile.qrc">
             <normaloff>:/new/icons/icons/stop.png</normaloff>:/new/icons/icons/stop.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="trwConc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wdgVerse" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="lblVerse">
            <property name="styleSheet">
             <string notr="true">background-color : white; color : black;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QToolButton" name="tbPrevVerse">
              <property name="toolTip">
               <string>previous verse</string>
              </property>
              <property name="text">
               <string>previous verse</string>
              </property>
              <property name="icon">
               <iconset resource="resfile.qrc">
                <normaloff>:/new/icons/icons/uparrow.png</normaloff>:/new/icons/icons/uparrow.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tbNextVerse">
              <property name="toolTip">
               <string>next verse</string>
              </property>
              <property name="text">
               <string>next verse</string>
              </property>
              <property name="icon">
               <iconset resource="resfile.qrc">
                <normaloff>:/new/icons/icons/downarrow.png</normaloff>:/new/icons/icons/downarrow.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbAcc">
              <item>
               <property name="text">
                <string>vowels</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>consonants</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>accents</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>morphology</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="tbFontPlus">
              <property name="toolTip">
               <string>larger font</string>
              </property>
              <property name="text">
               <string>larger font</string>
              </property>
              <property name="icon">
               <iconset resource="resfile.qrc">
                <normaloff>:/new/icons/icons/loupe_plus.png</normaloff>:/new/icons/icons/loupe_plus.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tbFontMinus">
              <property name="toolTip">
               <string>smaller font</string>
              </property>
              <property name="text">
               <string>smaller font</string>
              </property>
              <property name="icon">
               <iconset resource="resfile.qrc">
                <normaloff>:/new/icons/icons/loupe_minus.png</normaloff>:/new/icons/icons/loupe_minus.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="tbVerseClose">
              <property name="toolTip">
               <string>hide</string>
              </property>
              <property name="text">
               <string>hide</string>
              </property>
              <property name="icon">
               <iconset resource="resfile.qrc">
                <normaloff>:/new/icons/icons/stop.png</normaloff>:/new/icons/icons/stop.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2"/>
 <customwidgets>
  <customwidget>
   <class>CLatCopt</class>
   <extends>QWidget</extends>
   <header>latcopt.h</header>
   <container>1</container>
   <slots>
    <signal>query()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>CBookTextBrowser</class>
   <extends>QWidget</extends>
   <header>booktextbrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
